x/feemarket/types/params.go

```diff
@@ -18,8 +18,12 @@ var (
 	ParamStoreKeyBaseFee                  = []byte("BaseFee")
 	ParamStoreKeyEnableHeight             = []byte("EnableHeight")
 	ParamStoreKeyMinGasPrice              = []byte("MinGasPrice")
+	ParamStoreKeyMinGasMultiplier         = []byte("MinGasMultiplier")
 )
 
+// DefaultMinGasMultiplier is 0.5 or 50%
+var DefaultMinGasMultiplier = sdk.NewDecWithPrec(50, 2)
```

Comment:
we should add the same for `DefaultMinGasPrice` and replace the defaults

```suggestion
var DefaultMinGasMultiplier = sdk.NewDecWithPrec(50, 2)
// DefaultMinGasPrice is 0 (i.e disabled)
var DefaultMinGasPrice = sdk.ZeroDec()
```